Disable attempts by the build script to fetch missing tools.

--- a/build.py
+++ b/build.py
@@ -21,8 +21,12 @@
 import re
 import shutil
 import subprocess
-import requests
-from requests.exceptions import HTTPError
+
+#import requests
+#from requests.exceptions import HTTPError
+requests = None
+HTTPError = None
+
 import traceback
 from io import BytesIO
 import bz2
